bots/observability/tracing.py

```
@@ -0,0 +1,180 @@
+
+    Returns:
+        Tracer: OpenTelemetry tracer instance
+    """
+    # Ensure tracing is set up
+    if not _initialized:
+        setup_tracing()
+
+    return trace.get_tracer(name)
+
+
+def configure_exporter(exporter_type: str = None, exporter: SpanExporter = None, **kwargs) -> None:
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)